eg/eg_exec.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 import argparse
-import eg_config
-import eg_util
+from eg import eg_config
+from eg import eg_util
```

Comment:
In Python 3 It was throwign import errorw when installed by pip. I could add a try except for the import errors.